tests/test_pipeline.py

```diff
@@ -1,19 +1,24 @@
 import unittest
+import base64
 
 from guacml.pipeline import Pipeline
 from tests.test_util import load_dataset, read_fixture
 
 
 class TestPipeline(unittest.TestCase):
 
-    def test_predict(self):
+    def quick_run(self):
```

Comment:
The `quick_run` would be even quicker if we created a smaller tree to begin with rather then first creating the full thing and then removing steps and models. ;-)

But more importantly should we test the other models as well or do you think focussing only on XGBoost is good enough?